scene/gui/flow_container.cpp

```diff
@@ -152,6 +152,28 @@ void FlowContainer::_resort() {
 			line_data = lines_data[current_line_idx];
 		}
 
+		// The first child of each line adds the offset caused by the aligment,
+		// but only if the line doesn't contain a child that expands
```

Comment:
[Comments are supposed to end with a period](https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#comment-style-guide).
```suggestion
		// The first child of each line adds the offset caused by the alignment,
		// but only if the line doesn't contain a child that expands.
```